server/Mac/mf_event.h

```diff
@@ -18,7 +18,6 @@
  */
 
 #ifndef __MF_EVENT_H
-#define __XMF_EVENT_H
```

Comment:
Yeah - even though the old one was wrong (__XMF instead of __MF) we should fix it by correctly naming the #define and not just deleting it - and rename it as @akallabeth suggests